doc/release/1.4.0-notes.rst

```diff
@@ -24,12 +24,49 @@ For running on PyPy, PyPy3 6.0+ and NumPy 1.15.0 are required.
 Highlights of this release
 --------------------------
 
--
+- a new submodule, `scipy.fft`, now supersedes `scipy.fftpack`; this
+means support for ``long double`` transforms, faster multi-dimensional
+transforms, improved algorithm time complexity, release of the global
+intepreter lock, and control over threading behavior
+- substantial improvement to the documentation and functionality of
+several `scipy.special` functions, and some new additions
+- the generalized inverse Gaussian distribution has been added to
+`scipy.stats`
+- an implementation of the Edmonds-Karp algorithm in
+`scipy.sparse.csgraph.maximum_flow`
+- `scipy.spatial.SphericalVoronoi` now supports n-dimensional input, 
+has linear memory complexity, improved performance, and
+supports single-hemisphere generators
 
 
 New features
 ============
 
+`scipy.constants` improvements
+------------------------------
+`scipy.constants` has been updated with the CODATA 2018 constants.
+
+
+`scipy.fft` added
+-----------------
+scipy.fft is a new submodule that supersedes the scipy.fftpack submodule. For the most part, this is a drop-in replacement for numpy.fft and scipy.fftpack alike. With some important differences, scipy.fft:
+- uses NumPy's conventions for real transforms (rfft). This means the return value is a complex array, half the size of the full fft output. This is different from the output of fftpack which returned a real array representing complex components packed together.
+- the inverse real to real transforms (idct and idst) are normalized for norm=None in the same way as ifft. This means the identity idct(dct(x)) == x is now true for all norm modes.
```

Comment:
need to add a bunch of double backticks in this section

Reply:
Done, thanks